.github/workflows/auto_label.yml

```
@@ -0,0 +1,44 @@
+name: Label PRs & Issues
+
+on:
+  issues:
+    types: [opened, edited]
+  pull_request_target:
+    types: [edited, opened, synchronize, reopened]
+
+jobs:
+  auto-label-rocm:
+    runs-on: ubuntu-18.04
+    steps:
+    - name: Retrieve information
+      id: vars
+      run: |
+        set -eux
+        IS_PR=${{ github.event.pull_request }}
+        if [[ -n "${IS_PR}" ]]; then
+          TITLE="${{ github.event.pull_request.title }}"
+          ISSUE_NUMBER=${{ github.event.pull_request.number }}
+        else
+          TITLE="${{ github.event.issue.title }}"
+          ISSUE_NUMBER=${{ github.event.issue.number }}
+        fi
+        echo ::set-output name=TITLE::${TITLE}
+        echo ::set-output name=ISSUE_NUMBER::${ISSUE_NUMBER}
+        echo ::set-output name=OWNER::"${{ github.repository_owner }}"
+        echo ::set-output name=REPO::"${{ github.event.repository.name }}"
+    - name: Auto-label ROCm
+      run: |
+        set -eux
+        if [[ ${TITLE,,} == *rocm* ]]; then
+          curl \
+            -X POST \
+            -H "Authorization: token ${GITHUB_TOKEN}" \
+            https://api.github.com/repos/${OWNER}/${REPO}/issues/${ISSUE_NUMBER}/labels \
```

probably put quotes here to be safe
```suggestion
            "https://api.github.com/repos/${OWNER}/${REPO}/issues/${ISSUE_NUMBER}/labels" \
```